avalon/tools/workfiles/app.py

```diff
@@ -195,12 +198,17 @@ def setup(self, root):
             "comment": ""
         }
 
-        self.template = "{task[name]}_v{version:0>4}<_{comment}>"
-        templates = self.data["project"]["config"]["template"]
-        if "workfile" in templates:
-            self.template = templates["workfile"]
+        try:
+            self.template = os.environ["AVALON_WORKFILE_TEMPLATE"]\
```

Comment:
Where does `AVALON_WORKFILE_TEMPLATE` come from? This normally resides in the database.